examples/reordered-preconditioned-solver/reordered-preconditioned-solver.cpp

```diff
@@ -0,0 +1,111 @@
+// SPDX-FileCopyrightText: 2017-2023 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+
+#include <ginkgo/ginkgo.hpp>
+
+
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <string>
+#include "ginkgo/core/matrix/permutation.hpp"
```

Comment:
```suggestion
#include <string>


#include "ginkgo/core/matrix/permutation.hpp"
```
or remove this header